includes/widgets/class-wc-widget-price-filter.php

```diff
@@ -120,21 +120,14 @@ public function widget( $args, $instance ) {
 			$form_action = preg_replace( '%\/page/[0-9]+%', '', home_url( trailingslashit( $wp->request ) ) );
 		}
 
-		echo '<form method="get" action="' . esc_url( $form_action ) . '">
-			<div class="price_slider_wrapper">
-				<div class="price_slider" style="display:none;"></div>
-				<div class="price_slider_amount" data-step="' . esc_attr( $step ) . '">
-					<input type="text" id="min_price" name="min_price" value="' . esc_attr( $current_min_price ) . '" data-min="' . esc_attr( $min_price ) . '" placeholder="' . esc_attr__( 'Min price', 'woocommerce' ) . '" />
-					<input type="text" id="max_price" name="max_price" value="' . esc_attr( $current_max_price ) . '" data-max="' . esc_attr( $max_price ) . '" placeholder="' . esc_attr__( 'Max price', 'woocommerce' ) . '" />
-					<button type="submit" class="button">' . esc_html__( 'Filter', 'woocommerce' ) . '</button>
-					<div class="price_label" style="display:none;">
-						' . esc_html__( 'Price:', 'woocommerce' ) . ' <span class="from"></span> &mdash; <span class="to"></span>
-					</div>
-					' . wc_query_string_form_fields( null, array( 'min_price', 'max_price', 'paged' ), '', true ) . '
-					<div class="clear"></div>
-				</div>
-			</div>
-		</form>'; // WPCS: XSS ok.
+		wc_get_template( 'content-widget-price-filter.php', array(
```

Comment:
Can you please fix the coding standard violations? See https://travis-ci.org/woocommerce/woocommerce/jobs/522007335#L625

It is fine to ignore this violation https://travis-ci.org/woocommerce/woocommerce/jobs/522007335#L649. See https://github.com/woocommerce/woocommerce-sniffs/pull/5

Reply:
Done. The one in L649 came from the original code, though.

```
		echo '<form method="get" action="' . esc_url( $form_action ) . '">
			<div class="price_slider_wrapper">
				<div class="price_slider" style="display:none;"></div>
				<div class="price_slider_amount" data-step="' . esc_attr( $step ) . '">
					<input type="text" id="min_price" name="min_price" value="' . esc_attr( $current_min_price ) . '" data-min="' . esc_attr( $min_price ) . '" placeholder="' . esc_attr__( 'Min price', 'woocommerce' ) . '" />
					<input type="text" id="max_price" name="max_price" value="' . esc_attr( $current_max_price ) . '" data-max="' . esc_attr( $max_price ) . '" placeholder="' . esc_attr__( 'Max price', 'woocommerce' ) . '" />
					<button type="submit" class="button">' . esc_html__( 'Filter', 'woocommerce' ) . '</button>
					<div class="price_label" style="display:none;">
						' . esc_html__( 'Price:', 'woocommerce' ) . ' <span class="from"></span> &mdash; <span class="to"></span>
					</div>
					' . wc_query_string_form_fields( null, array( 'min_price', 'max_price', 'paged' ), '', true ) . '
					<div class="clear"></div>
				</div>
			</div>
		</form>'; // WPCS: XSS ok.
```

Reply:
Thanks